docs/logging-and-managing-experiment-results/index.rst

```diff
@@ -2,101 +2,153 @@
 
 Logging and managing experiment results
 =======================================
+Log and manage data in Neptune via |neptune-client-github| (Python and R clients are available).
 
-|Youtube Video|
+[loom-placeholder]
 
-|neptune-client-github|
-is an open source Python library that lets you integrate your Python scripts with Neptune so that you can more easily track and organize your experiments in the rich Neptune |dashboard|.
+What is neptune-client?
+-----------------------
+|neptune-client-github| is an open source Python library that serves two purposes:
 
-Once you have integrated with Neptune, you can also:
+#. logging machine learning experiments,
+#. update existing experiment with new data and visualizations,
+#. download experiment data from Neptune to local machine.
 
-* Create experiments. |Example1|.
-* Manage running experiments. |Example2|.
-* Fetch experiment and project data. |Example3|.
+It is designed to be:
 
-**Example**
+* **lightweight**: low setup effort,
+* **generic**: capable of logging any kind of machine learning work
+* **straightforward**: user defines what to keep track of during experiment to use.
 
-The following code creates a Neptune experiment in the project |onboarding|. Name (*Python str*) and parameters (*Python dict*) are added to the experiment in the ``create_experiment()`` method. The code logs ``iteration``, ``loss`` and ``text_info`` metrics to Neptune in real time, using three dedicated methods. It also showcases a common use case for Neptune client, that is, tracking progress of machine learning experiments.
+You can :ref:`log experiments from anywhere <execution-environments-index>` (local machine, cluster, cloud, Colab, etc.) and they will be tracked in the same, standardized way. You will be able to quickly compare experiments run by you on your workstation with experiments run on AWS by your team-mate.
 
+What you can do?
```

Comment:
looking at it now this feels like a repetition -> please merge this section with the one above.

Reply:
ok